ErrorConvert is not implemented for VerboseError

## Prerequisites

Here are a few things you should provide to help me understand the issue:

- Rust version : `rustc 1.47.0 (18bf6b4f0 2020-10-07)`
- nom version : `6.0.0`
- nom compilation features used:

## Test case

Example test case:

```rust
use nom::{
    bits,
    bits::complete::take,
    error::{context, VerboseError},
    IResult,
};

fn parse(input: &[u8]) -> IResult<&[u8], u8, VerboseError<&[u8]>> {
    let take_two = |i| context("parse", take::<_, _, _, VerboseError<_>>(2usize))(i);
    bits(take_two)(input)
}

fn main() {
    let input = b"hello";
    assert_eq!(parse(b"hello"), Ok((&input[1..], 1u8)));
}
```

Error message is:

```
error[E0277]: the trait bound `nom::error::VerboseError<(_, usize)>: nom::ErrorConvert<_>` is not satisfied
  --> src/main.rs:10:10
   |
10 |     bits(take_two)(input)
   |          ^^^^^^^^ the trait `nom::ErrorConvert<_>` is not implemented for `nom::error::VerboseError<(_, usize)>`
   |
  ::: /nom-6.0.0/src/bits/mod.rs:36:48
   |
36 | pub fn bits<I, O, E1: ParseError<(I, usize)> + ErrorConvert<E2>, E2: ParseError<I>, P>(
   |                                                ---------------- required by this bound in `nom::bits`
```

--- a/src/traits.rs
+++ b/src/traits.rs
@@ -1609,6 +1609,26 @@ impl<I> ErrorConvert<error::Error<(I, usize)>> for error::Error<I> {
   }
 }
 
+#[cfg(feature = "alloc")]
+#[cfg_attr(feature = "docsrs", doc(cfg(feature = "alloc")))]
+impl<I> ErrorConvert<error::VerboseError<I>> for error::VerboseError<(I, usize)> {
+  fn convert(self) -> error::VerboseError<I> {
+    error::VerboseError {
+      errors: self.errors.into_iter().map(|(i, e)| (i.0, e)).collect(),
+    }
+  }
+}
+
+#[cfg(feature = "alloc")]
+#[cfg_attr(feature = "docsrs", doc(cfg(feature = "alloc")))]
+impl<I> ErrorConvert<error::VerboseError<(I, usize)>> for error::VerboseError<I> {
+  fn convert(self) -> error::VerboseError<(I, usize)> {
+    error::VerboseError {
+      errors: self.errors.into_iter().map(|(i, e)| ((i, 0), e)).collect(),
+    }
+  }
+}
+
 #[cfg(test)]
 mod tests {
   use super::*;

